tests/testUtils.py

```diff
@@ -205,7 +205,8 @@ def checkDelayedOutput(popen, cmd, ignoreError=False):
         Utils.checkOutputFileWrite(start, cmd, output, error)
         if popen.returncode != 0 and not ignoreError:
             Utils.Print("ERROR: %s" % error)
-            raise subprocess.CalledProcessError(returncode=popen.returncode, cmd=cmd, output=error)
+            # for now just include both stderr and stdout in output, python 3.5+ supports both a stdout and stderr attributes
```

Comment:
what is the Python 3.5+ comment here getting at -- even Ubuntu 18 is using 3.6 so if there is a 3.5+ feature we should be able to use it

Reply:
`stderr` was being placed in `output`. Python 3.5+ has both an `output` and a `stderr`. The better fix is to put `stdout` into `output` and `stderr` into `stderr`. However, `output` is used in 20+ places in many tests. I didn't want to make that big of a change in `release/3.1`. In `main` I plan to sperate out the `stdout` and `stderr` and update the 20+ usages of the `CalledProcessError`.